assets/tests/components/detours/diversionPage.test.tsx

```diff
@@ -1039,13 +1029,16 @@ describe("DiversionPage", () => {
     const { container } = render(
       <DiversionPage
         originalRoute={{
-          routeName,
-          routeOrigin,
-          routeDescription,
-          routeDirection,
-          shape: shapeFactory.build({
-            points: [connectionPoint],
-          }),
+          route: {
+            name: routeName,
+          },
+          routePattern: {
+            headsign: routeDescription,
+            name: routeOrigin,
+            shape: {
+              points: [connectionPoint],
+            },
+          },
```

Comment:
note: unsure about this due to how encapsulation is broken here.

Reply:
IMO this is fine. The fact that `routedDescription`, for instance, comes from the `headsign` of the `routePattern` is a meaningful statement about how `<DiversionPage />` works, so it's not wrong to have the test be explicit about that here.